contributors/design-proposals/architecture/scope.md

```diff
@@ -0,0 +1,342 @@
+# Kubernetes scope
+
+Purpose of this doc: Clarify factors affecting decisions regarding
+what is and is not in scope for the Kubernetes project.
+
+Related documents:
+* [What is Kubernetes?](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/)
+* [Kubernetes design and architecture](architecture.md)
+* [Kubernetes architectural roadmap (2017)](architectural-roadmap.md)
+* [Design principles](principles.md)
+* [Kubernetes resource management](resource-management.md)
+
+Kubernetes is a portable, extensible open-source platform for managing
+containerized workloads and services, that facilitates both
+declarative configuration and automation.  Workload portability is an
+especially high priority. Kubernetes provides a flexible, easy-to-run,
+secure foundation for running containerized applications on any cloud
```

Comment:
nit: public cloud provider

Reply:
So you think we're talking about three things here?  (Public cloud, private cloud), and "your own systems"?  I was thinking "public cloud" and "private/your own systems".  I would understand either approach, but I think right now the wording is confusing, and contributes to the existing confusion where many people think that "cloud" implicitly means "public cloud", which in my mind it does not.

Reply:
"Your own systems" could also be your laptop, pair of workstations under your desk, raspberry pi, or whatever.

This came verbatim out of an email thread, so I'm happy to tweak, but I don't think this particular point is worth a lot of wordsmithing.